File: Basics/For Loop/Exercise/08.py
number_tournaments = int(input())
starting_points = int(input())

points_earned = 0
wins = 0


for tournament in range(number_tournaments):
    position = input()

    if position == 'W':
        points_earned += 2000
        wins += 1

    elif position == 'F':
        points_earned += 1200

    elif position == 'SF':
        points_earned += 720

total_point = starting_points + points_earned
print(f"Final points: {total_point}")
print(f"Average points: {points_earned // number_tournaments}")
print(f"{(wins / number_tournaments * 100):.2f}%")

File: JS-Basics/For Loop/Exercise/08.js
function calculateTennisStats(input) {
    const tournamentCount = parseInt(input[0]); 
    const startingPoints = parseInt(input[1]);
    const results = input.slice(2); 

    let totalPoints = startingPoints; 
    let pointsEarned = 0;
    let wins = 0; 

    for (const result of results) {
        if (result === "W") {
            pointsEarned += 2000;
            wins++;
        } else if (result === "F") {
            pointsEarned += 1200;
        } else if (result === "SF") {
            pointsEarned += 720;
        }
    }

    totalPoints += pointsEarned; 
    const averagePoints = Math.floor(pointsEarned / tournamentCount);
    const winPercentage = (wins / tournamentCount * 100).toFixed(2); 

    console.log(`Final points: ${totalPoints}`);
    console.log(`Average points: ${averagePoints}`);
    console.log(`${winPercentage}%`);
}
